### ISSUE TITLE
Can't install directly from GitHub

### ISSUE THREAD

**Issue by LykaiosNZ** [alsbury/spice2json#2]
Trying to install directly from the repo results in an error due to the module name not matching the repo

```
>go install github.com/alsbury/spice2json@latest
go: github.com/alsbury/spice2json@latest: version constraints conflict:
        github.com/alsbury/spice2json@v0.3.2: parsing go.mod:
        module declares its path as: spice2json
                but was required as: github.com/alsbury/spice2json
```

Obviously it can be installed by cloning the repo and running `go install` from the repo root but it would be nice if it could be installed directly.

**Comment by alsbury** [comment]
@LykaiosNZ this was my first and only Go project, so I'm not all that familiar with the ecosystem and how it all works. I updated the module name and created a new version tag and release. Then I ran `GOPROXY=proxy.golang.org go list -m github.com/alsbury/spice2json@v0.3.3`. I think that I'm waiting for my package to get indexed on `pkg.go.dev`. If you know of any other improvements to make the package accessible, feel free to create a PR.
